soccer/gameplay/skills/line_kick.py

```diff
@@ -44,6 +45,13 @@ def __init__(self):
         self.max_speed = None
         self.max_accel = None
 
+    def on_enter_waiting(self):
+        super().recalculate_aim_target_point()
```

Comment:
long term we need to use move_to if we are going to keep this

Reply:
We should just make one above that uses both the move and line kick subbehavior. You dont what this move in there when there are moving balls and that.